build.gradle

```diff
@@ -690,7 +690,6 @@ startScripts {
     "-XX:G1ConcRefinementThreads=2",
     "-XX:G1HeapWastePercent=15",
     "-XX:MaxGCPauseMillis=100",
-    "-XX:StartFlightRecording,dumponexit=true,settings=default.jfc",
```

Comment:
I recommend to keep it but remove only dumponexit=true, to not dump the file when the process stops or crashes.